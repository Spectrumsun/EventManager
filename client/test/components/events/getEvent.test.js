import React from 'react';
import { shallow } from 'enzyme';
import Display from '../../../src/components/Event/getEvent';

const props = {
  eventName: 'tome',
  startDate: '2018-10-10',
  endDate: '2018-11-12',
  purpose: 'lol',
  centerImage: 'http://lol/com',
};

describe('<Display />', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Display {...props} />);
  });

  it('should render the Display', () => {
    shallow(<Display {...props} />);
  });

  it('should render initial layout of Display', () => {
    expect(wrapper.getElements()).toMatchSnapshot();
  });
  
  it('should have image on layout', () => {
    expect(wrapper.find('img').length).toEqual(1);
  });
});
