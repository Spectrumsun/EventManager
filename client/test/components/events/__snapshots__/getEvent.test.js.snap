// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> should render initial layout of Display 1`] = `
Array [
  <div
    className="card d-lg-inline-block"
    style={
      Object {
        "width": "20rem",
      }
    }
  >
    <img
      alt="Card cap"
      className="card-img-top"
      src="http://lol/com"
      style={
        Object {
          "height": "15rem",
          "objectFit": "cover",
        }
      }
    />
    <div
      className="card-body"
    >
      <h4
        className="card-title"
      >
        tome
      </h4>
      <span
        className="card-text"
      >
        2018-10-10
      </span>
       To 
      <span
        className="card-text"
      >
        2018-11-12
      </span>
      <h6
        className="card-text"
      >
        lol
      </h6>
      <h6
        className="btn btn-primary"
      >
        View more
      </h6>
    </div>
  </div>,
]
`;
